### DISCUSSION TITLE
[Python] Call GDL from Python 3.8

### DISCUSSION THREAD

**Discussion by Minnielam** [gnudatalanguage/gdl#881] [General]
Hi all, 

Can I ask a question about calling GDL in Python environment? I followed the previous tutorial but it failed.  In fact, GDL worked well independently but it could not call in Python. Besides, I could import the GDL package.

The following is my error messages:

>>> import GDL
>>> arg = 2**16 -1
>>> retval = GDL.function('/Users/xxxx/GDL/test_func.pro',arg)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'GDL' has no attribute 'function'

Do you have any ideas of this issue? Is that any documents/ User Manual regarding this question? Many thanks.

**Comment by slayoo** [comment]
I believe that the best what we have in terms of exemplifying the "GDL from Python" bridge is here:
https://github.com/gnudatalanguage/gdl/blob/master/testsuite/python/test-GDL.py

Does this test run correctly on your installation (it is intended to be run via pytest)?

This test is part of continuous integration runs and IIRC comes from the Debian python-gdl package https://packages.debian.org/python-gdl (thanks to @olebole).

**Reply by Minnielam** [reply]
Thanks a lot @slayoo. I checked my cmake command and I did -DPYTHON=off due to the anaconda conflict. I will try to specify the Python directory and rerun cmake.
